indico/modules/events/editing/client/js/components/Timeline/judgment/UpdateFilesForm.jsx

```diff
@@ -26,20 +26,34 @@ import FinalTagInput from './TagInput';
 
 import './JudgmentBox.module.scss';
 
+const confirmOptions = [
+  {
+    value: 'confirm',
+    text: Translate.string('Confirm'),
+  },
+  {
+    value: 'confirm_and_approve',
+    text: Translate.string('Confirm & Approve'),
+  },
+];
+
 export default function UpdateFilesForm({setLoading}) {
+  const [confirmationType, setConfirmationType] = useState('confirm');
+  const {fileTypes} = useSelector(selectors.getStaticData);
   const lastRevision = useSelector(selectors.getLastRevision);
   const staticData = useSelector(selectors.getStaticData);
   const {eventId, contributionId, editableType} = staticData;
-  const fileTypes = useSelector(selectors.getFileTypes);
   const dispatch = useDispatch();
   const files = getFilesFromRevision(fileTypes, lastRevision);
+  const option = confirmOptions.find(x => x.value === confirmationType);
 
   const submitReview = async formData => {
     setLoading(true);
     const rv = await dispatch(
       reviewEditable(lastRevision, {
         ...formData,
         action: EditingReviewAction.update,
+        accepted_with_changes: confirmationType === 'confirm_and_approve',
```

Comment:
I would go for `accepted_with_changes`.

Reply:
to me `accepted_with_changes` sounds more like the name of a state.

PS: Another option would be to add a new `EditingReviewAction` (`update_accept` or something like that) and use that instead of the separate flag. Not sure in how many places we'd have to add logic to handle it though...